tests/test_pytorch/test_profiler.py

```diff
@@ -280,3 +280,9 @@ def test_hpu_trace_event_kernel(tmpdir):
             raise Exception("Could not find event kernel in trace")
         for event_duration in event_duration_arr:
             assert event_duration >= 0
+
+
+def test_hpu_profiler_env(monkeypatch):
```

Comment:
is there any profiler test that we need to enable back ?

Reply:
enabled